object_files_api/files_api.py

```diff
@@ -102,7 +103,12 @@ def _save_json_to_dynamo(self, files_json: dict) -> bool:
         files_json = _serialize_json(files_json)
         if 'expireTime' not in files_json:
             files_json['expireTime'] = _get_expire_time(3)
-        self.files_table.put_item(Item=files_json)
+        try:
+            self.files_table.put_item(Item=files_json)
```

Comment:
Just a suggestion. I'd remove this from the constructor
```
        if not self.event['local']:
            self.dynamodb = boto3.resource('dynamodb')
            self.files_table = self.dynamodb.Table(self.config['files-tablename'])
```
add this to the imports:
```
from botocore.exceptions import ClientError
```
and put that update this method _save_json_to_dynamo
```
        try:
            files_table = boto3.resource('dynamodb').Table(self.config['files-tablename'])
            files_table.put_item(Item=files_json)
        except ClientError as ce:
            success_flag = False
            capture_exception(ce)
            print(f"Error saving to {self.config['files-tablename']} table - {ce.response['Error']['Code']} - {ce.response['Error']['Message']}")
```

I like this flow because all db operations and error handling are encapsulated here. You're currently not doing any error handling in the constructor which could fail if the database does not exist. However there _could_ be a performance hit(idk) since it'd be calling "boto3.resource('dynamodb').Table" every time this method would be called. You'd have to test it under load since I have no idea.